src/components/ImageUpload/index.jsx

```diff
@@ -132,6 +140,7 @@ ImageUpload.propTypes = {
 
 ImageUpload.defaultProps = {
   className: '',
+  disabled: false,
```

Comment:
Just a reminder to update this if you end up updating the default value in the form.

Reply:
I still think this is a fine default. This is going to be overturned by parent component all the time (which now defaults to disabled=True). As a component-level default, I think being enabled by default makes sense.